Jenkinsfile

```diff
@@ -65,6 +65,12 @@ elifePipeline {
                     return m
                 })
             }
+
+            stage 'Trigger grobid and grobid trainer tag update', {
+                def grobidTag = allGrobidTags.last()
+                def grobidTrainerTag = "${grobidTag}-${commit}"
+                build job: 'dependencies/dependencies-sciencebeam-trainer-delft-update-grobid-and-trainer',  wait: false, parameters: [string(name: 'grobid_tag', value: grobidTag), string(name: 'grobid_trainer_tag', value: grobidTrainerTag)]
```

Comment:
very minor: just wondering whether we can break that line up (in terms of line length)